x-pack/plugins/apm/public/components/shared/charts/CustomPlot/index.js

```diff
@@ -234,13 +234,16 @@ InnerCustomPlot.propTypes = {
       firstSeen: PropTypes.number,
     })
   ),
+  noHits: PropTypes.bool,
```

Comment:
Oh man. How I do not miss prop types. We should just byte the bullet and move to elastic charts asap.

Reply:
I totally agree, `CuromPlot` and `Histogram` are overcomplicated and not tested.